src/authentication/accept-invite/styles.scss

```diff
@@ -0,0 +1,93 @@
+@use '~@zero-tech/zui/styles/theme' as theme;
+
+.invite {
+  &__title {
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 36px;
+    line-height: 0px;
+    margin-bottom: 21px;
+    margin-top: 90px;
+
+    text-align: center;
+    text-shadow: 0px 0px 10px rgba(255, 255, 255, 0.7);
+    color: theme.$color-greyscale-12;
+  }
+
+  &__text {
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    text-align: center;
+    color: theme.$color-greyscale-11;
+    line-height: 0px;
+    margin-bottom: 28px;
+  }
+
+  &__input {
+    width: 284px;
+    height: 40px;
+
+    background: theme.$color-greyscale-transparency-3;
+    border: theme.$color-greyscale-7;
```

Comment:
There is difference when i remove the `background` prop. No difference when i remove the border property (so i removed that one)

without background
<img width="356" alt="image" src="https://user-images.githubusercontent.com/33264364/234028406-0866b20d-99df-4147-8159-c3f7acc7c7b0.png">

with background

<img width="387" alt="image" src="https://user-images.githubusercontent.com/33264364/234028273-7e06b1d8-095a-45f7-bdd7-550ce16c6867.png">

Reply:
Yeah, we might want to confirm with design that this was intended. It doesn't make sense to me that we'd want custom styling on our common input. This doesn't seem to match any of the real input samples in the design. But, again, this can be a follow up.

Reply:
That's the idea, yeah. That way, if they ever change the design of the theme, every input would change automatically. It's meant to simplify our usage too. We should be able to just use the components with very little thought about style aside from some positioning/sizing perhaps.